### DISCUSSION TITLE
Trying to import from stable but folder is on external drive

### DISCUSSION THREAD

**Discussion by integerrr** [ppy/osu#16160] [Q&A] [answered]
Where should I symlink the stable osu folder to on my system? The partition in question is NTFS and lazer (on macos) doesnt seem to want to show contents of the folder when it prompts me to locate the stable installation:
<img width="894" alt="image" src="https://user-images.githubusercontent.com/7279624/146685290-d56feb20-66aa-4f76-b56b-153665e98f23.png">

**Comment by bdach** [comment]
Given the stable directory selector I'd like to think it shouldn't matter where you symlink the directory, as long as you then select it on the selector?

Prior to the selection existing the instructions on how to symlink were thus: https://github.com/ppy/osu/issues/3763#issuecomment-769908256

**Reply by integerrr** [reply]
> Given the stable directory selector I'd like to think it shouldn't matter where you symlink the directory, as long as you then select it on the selector?

Yes thank you I dont know why this didnt occur to me when I wrote the post

I symlinked it initially as ~/osu! but somehow the game went 
<img width="426" alt="image" src="https://user-images.githubusercontent.com/7279624/146687897-2f31e8c5-5058-4c18-9595-17e9adc572b7.png">
so i read the linked post and redid the symlink as ~/.osu as suggested, as well as deleted the previous symlink, predictably it's now throwing this:
<img width="427" alt="image" src="https://user-images.githubusercontent.com/7279624/146687934-54c75aa9-fa39-49d4-81e4-05c85c4a19a5.png">

where can I manually change the directory the game reads from?

**Reply by bdach** [reply]
As far as I can see the path to the directory selected isn't saved anywhere and a game restart should be sufficient for the select dialog to be shown again.

**Reply by integerrr** [reply]
Thanks, it's still saying "no skins were found to import" tho. Even score and map imports are saying the same thing.
[logs.zip](https://github.com/ppy/osu/files/7741681/logs.zip)

**Reply by bdach** [reply]
Interestingly `database.log` is not in the zip above. Was it forgotten, or is it in fact missing?

**Reply by integerrr** [reply]
It wasn't there

**Comment by peppy** [comment, marked as answer]
If it's not showing on the selector, I think you need to figure why that is the case first. I don't think symlinking is going to help you at this point. It's probably whatever software you're using to get NTFS support doing something weird.

As an example, here's an external drive working fine for me:

![20211220 172032 (osu!)](https://user-images.githubusercontent.com/191335/146735168-4ff66dab-a0a5-4c7c-8525-57abc2a1d29e.png)

**Reply by integerrr** [reply]
Fwiw, this was a fresh OS install so there likely wouldn't have been anything that can affect NTFS reads, I could get ffmpeg to read files from the partition through terminal so I'm not sure about that and afaik macos can read NTFS natively?

**Reply by peppy** [reply]
if you're doing reads only then it can, i believe. i'll see if i can test this out locally.

**Reply by peppy** [reply]
I can reproduce nothing showing up with the same drive as above, formatted to NTFS.

**Reply by peppy** [reply]
Tracking at https://github.com/ppy/osu/issues/16193.

**Reply by integerrr** [reply]
Interestingly, this works on Catalina

**Reply by integerrr** [reply]
Wondering how i can get around this for stable beatmap import before this can be fixed?

**Reply by peppy** [reply]
Copy the files to `~/.osu` and run import.
